roles/openshift_control_plane/tasks/pre_pull.yml

```diff
@@ -4,9 +4,8 @@
   register: control_plane_image
 
 # This task runs async to save time while the master is being configured
-- name: Pre-pull Origin image
-  docker_image:
-    name: "{{ osm_image }}"
+- name: Pre-pull Origin image (docker)
+  command: "{{ openshift_container_cli }} pull {{ osm_image }}"
```

Comment:
Will openshift_container_cli also support the following NO_PROXY environment setting?

Reply:
It should, yes. CI is not testing that so please report the issue if it breaks proxied environments